packages/loot-core/src/server/main.ts

```diff
@@ -391,6 +392,18 @@ handlers['category-group-move'] = mutator(async function ({ id, targetId }) {
   });
 });
 
+handlers['apply-multiple-templates'] = mutator(async function ({
```

Comment:
probably not needed since there don't need to be an "api" for this

Reply:
So you want me to just import the function from the goalTemplates ?

Reply:
The queries file has all the other examples of how the existing options work if we can match that.

Reply:
you want it to be triggered via a applyBudgetAction ?

Reply:
That would be nice since all the others are that way

Reply:
Got it. Tbh i dont get the actual structure yet so i just did it the way the other functions on the Group work :D